dist/redhat-brand-theme.css

```diff
@@ -1046,16 +1046,132 @@
   --pf-v6-c-skeleton--BorderRadius: var(--rh-border-radius-sharp, 0);
 }
 
-.pf-v6-c-modal-box {
-  --pf-v6-c-modal-box--BorderRadius: var(--rh-border-radius-sharp, 0px);
+:root {
+  --rh-color-light-gray: #fafafa;
+  --rh-color-gray-300: #d2d2d2;
+  --rh-color-brand-yellow: #f0ab00;
+  --rh-color-gray-700: #6a6e73;
+}
+
+:root:where(.pf-v6-theme-dark) {
+  --rh-color-light-gray: #fafafa;
+  --rh-color-gray-300: #d2d2d2;
+  --rh-color-brand-yellow: #f0ab00;
+  --rh-color-gray-700: #6a6e73;
+}
+
+.pf-v6-c-table {
+  /* Header styles */
+  --pf-v6-c-table__thead--cell--FontWeight: var(--pf-v5-c-table__thead--cell--FontWeight, 700);
```

Comment:
continuing to reference --pf-v5- css variables means you’ll need to continue shipping all of the PF5 css in addition to the PF6 CSS to get this theme to work. 

Is that unavoidable & will you always have the PF5 CSS in your environment? If the rest of the environment continues forward with modernization and migrations, you’ll have to rewrite all this CSS if you ever want to stop shipping PF5 styles.